lte/gateway/release/build-magma.sh

```diff
@@ -137,6 +137,7 @@ MAGMA_DEPS=(
     "nlohmann-json3-dev"
     "sentry-native"   # sessiond
     "td-agent-bit >= 1.7.8"
+    "bpfcc-tools" # required for kernsnoopd
```

Comment:
Ships an integrated clang. 63.0 MB in total. Where else do I put this dep?

Reply:
@tmdzk / @quentinDERORY on adding new dependencies

Reply:
Here https://github.com/magma/magma/blob/87603aff84a8dda0e75d362455f75b369972f849/lte/gateway/deploy/roles/magma/tasks/main.yml#L240

Reply:
Done.